templates/docs/patterns/accordion.md

```diff
@@ -36,6 +36,12 @@ View example of the accordion pattern
 View example of the accordion pattern
 </a></div>
 
+**With checkboxes:**
+
+<div class="embedded-example"><a href="/docs/examples/patterns/accordion/checkboxes/" class="js-example">
```

Comment:
Would be nice to rename the example file as well from `checkboxes` to `tick-elements`

Reply:
Oh, you did rename the file, but haven't updated the link here, so it errors in linkchecker.

Reply:
Yeah, sorry fixed. I pushed 3 times with tiny changes like this.